### PR TITLE
fix: the format of hover info in LG editor

### PR DESCRIPTION
**Pull request by cosmicshuai**
#minor

change the display info of hover more close to the original content:
image

### CODE REVIEW

File: Composer/packages/tools/language-servers/language-generation/src/LGServer.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ import isEqual from 'lodash/isEqual';
 import { filterTemplateDiagnostics, isValid, lgUtil } from '@bfc/indexers';
 import { MemoryResolver, ResolverResource, LgFile } from '@bfc/shared';
 import { buildInFunctionsMap } from '@bfc/built-in-functions';
+import { LgTemplate } from '@botframework-composer/types';
```
Comment (hatpick) [inline Composer/packages/tools/language-servers/language-generation/src/LGServer.ts:33]: add this dependency to package.json

File: Composer/packages/tools/language-servers/language-generation/src/LGServer.ts [hunk 2]
```diff
@@ -367,6 +373,18 @@ export class LGServer {
...
 
+  private buildHoverTemplateInfo(template: LgTemplate) {
+    let templateName = '';
+    if (template.parameters.length > 0) {
+      templateName = `\\# ${template.name}(${template.parameters.join(', ')})`;
+    } else {
+      templateName = `\\# ${template.name}`;
+    }
+
+    const templateBody = template.body.replace(/-/g, '\\-').replace(/\r?\n/g, '\n\n');
```
Comment (hatpick) [inline Composer/packages/tools/language-servers/language-generation/src/LGServer.ts:384]: this should only replace all the non-escaped ```-```

Comment (hatpick) [reply Composer/packages/tools/language-servers/language-generation/src/LGServer.ts:384]: also only replace new lines at the end of each line

**Review by hatpick — CHANGES_REQUESTED** [review]

File: Composer/packages/tools/language-servers/language-generation/src/LGServer.ts (hunk 1 above)
Comment (cosmicshuai) [reply Composer/packages/tools/language-servers/language-generation/src/LGServer.ts:33]: added

File: Composer/packages/tools/language-servers/language-generation/src/LGServer.ts (hunk 2 above)
Comment (cosmicshuai) [reply Composer/packages/tools/language-servers/language-generation/src/LGServer.ts]: I found I only need to keep the original content. So I use a code block to warp the template info here.

**Comment by hatpick** [comment]
@cosmicshuai you have broken tests, can you fix them?

**Comment by cosmicshuai** [comment]
@hatpick I just finished the broken tests.

**Review by hatpick — APPROVED** [review]

**Pull request merged**